core/store/models/log_events.go

```diff
@@ -254,6 +255,19 @@ func (le InitiatorLogEvent) JSON() (JSON, error) {
 	return out, json.Unmarshal(b, &out)
 }
 
+// Meta returns metadata about the ethereum log event
+func (le InitiatorLogEvent) Meta() JSON {
+	el := le.Log
+	meta := make(map[string]interface{})
+	meta["initiator"] = map[string]interface{}{
+		"transactionHash": el.TxHash,
+	}
```

Comment:
What about using an anonymous struct definition as mentioned before?

Reply:
I tried it and got a syntax error. Couldn't figure out how to make it work... any help would be appreciated :)

Reply:
Check up on anonymous struct, its quickly described here -> http://julianyap.com/2013/09/23/using-anonymous-structs-to-pass-data-to-templates-in-golang.html

Reply:
Thanks, it is possible to do this but to my eye it looks more confusing than the other version:

```
struct {
			Initiator struct {
				TransactionHash string
			}
		}{
			struct{ TransactionHash string }{jobRun.RunRequest.TxHash.Hex()},
		}
```